wordpress-mobile/WordPress-iOS
# Zendesk: Pass new request configuration to Help Center and My Tickets

Fixes #11014 

Somewhere along the way, Zendesk changed it's logic/flow so that tickets created from the Help Center and My Tickets list no longer used our custom new request information. This PR adheres to the new Zendesk design, which is to send the new request configuration to the Help Center and My Tickets controllers.

To test:
- Go to Me > Help & Support.
- Create tickets from these paths: (NOTE: don't use your Automattic email address. Ref #11508.)
  - Contact Us
  - Help Center > right button on nav bar
  - My Tickets > right button on nav bar
- Go to Zendesk and look at your tickets.
- Verify all 3 have the same tags (assuming you've used the same site all 3 times).
  - Tip: Zendesk itself adds some tags to tickets. To see tags added just by the app, in `ZendeskUtils.getTags`, log out the `tags` array that is returned.

Example ZD tags:
<img width="319" alt="zd_tags" src="https://user-images.githubusercontent.com/1816888/56541557-4fe54200-6529-11e9-8536-313b592aa75c.png">
